homeassistant/components/flux_led/select.py

```diff
@@ -202,3 +210,28 @@ async def async_select_option(self, option: str) -> None:
         """Change the remote config setting."""
         remote_config: RemoteConfig = self._name_to_state[option]
         await self._device.async_config_remotes(remote_config)
+
+
+class FluxWhiteChannelSelect(FluxConfigSelect):
```

Comment:
Need to check the base class is correct as we don't want coordinator updates since it can't change once its setup

Reply:
Fixed in b4f72ba7b9